cv.tex

```diff
@@ -29,9 +33,11 @@
 
 \lastupdated % Print the Last Updated text at the top right
 
-\namesection{Debarghya}{Das}{ % Your name
-\urlstyle{same}\url{http://debarghyadas.com} \\ % Your website, LinkedIn profile or other web address
-\href{mailto:dd367@cornell.edu}{dd367@cornell.edu} | 607.379.5733 % Your contact information
+\namesection{Joe}{Pate}{ % Your name
+\urlstyle{same}\url{https://linkedin.com/in/whoisjoepate/} \\ % Your website, LinkedIn profile or other web address
+\href{mailto:stress@lotsofmoving.parts}{stress@lotsofmoving.parts} | 302.440.4540 \\% Your contact information
+\sectionspace % Some whitespace after the section
+I'm a decadal Information Security practitioner with focus on security incident detection and response. I enjoy making and integrating tools to improve operational process.
```

Comment:
(JC):
The summary/objective is kind of heavy on the first line. Should you line-break sooner?

Reply:
I agree with this feedback, it took a few iterations to get this formatting which i think is passable but not great, wastes a bit more space than I'd like. 

Can't promise this will get addressed but i'll play with it

Reply:
played with it, leaving it how it is